sd_dynamic_prompts/dynamic_prompting.py

```diff
@@ -84,6 +86,17 @@ def get_magic_prompt_device() -> torch.device:
     return device
 
 
+def requote_prompt(prompt: str) -> str | None:
+    # if prompt is empty, return None
+    # if prompt will not be auto quoted by webui and can effected by stripping, we need to quote it ourselves
+    if prompt:
```

Comment:
You'd get less of a pyramid if you early-exited out here with `if not prompt: return None`